src/Hadrian/Package.hs

```diff
@@ -63,6 +63,7 @@ isCPackage _ = False
 -- | Is this a Haskell package?
 isHsPackage :: Package -> Bool
 isHsPackage (Package Haskell _ _ _) = True
+isHsPackage (Package _ _ "rts" _)   = True
```

Comment:
This is quite confusing, since `rts` is defined as `cTop "rts"`. This at least deserves an explanation in a comment.

Reply:
Fixed in an upcoming commit.